librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -498,6 +498,78 @@ commands:
       - name: type
         type: RZ_CMD_ARG_TYPE_NUM
         optional: true
+  - name: pv
+    summary: Print bytes based on current bitness and endianness
+    subcommands:
+      - name: pv
+        summary: print bytes
+        cname: print_value
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_JSON
+          - RZ_OUTPUT_MODE_RIZIN
+        args:
+          - name: repeat
+            type: RZ_CMD_ARG_TYPE_NUM
```

Comment:
```suggestion
            type: RZ_CMD_ARG_TYPE_RZNUM
```

Reply:
also in the other places

Reply:
`repeat` doesn't need to be resolved from flags or anything like that, just a number

Reply:
Done